Mathlib/Algebra/Homology/HomotopyCategory/DegreewiseSplit.lean

```diff
@@ -85,14 +85,13 @@ noncomputable def mappingConeHomOfDegreewiseSplitXIso (p q : ℤ) (hpq : p + 1 =
     have s_g := (σ (p + 1)).s_g
     have f_r := (σ (p + 1)).f_r
     dsimp at s_g f_r ⊢
-    simp? [mappingCone.ext_from_iff _ (p + 1) _ rfl, reassoc_of% f_r, reassoc_of% s_g] says
-      simp only [Cochain.ofHom_v, id_comp, comp_sub, sub_comp, assoc, reassoc_of% s_g,
-        ShortComplex.Splitting.s_r_assoc, ShortComplex.map_X₃, eval_obj, ShortComplex.map_X₁,
-        zero_comp, comp_zero, reassoc_of% f_r, zero_sub, sub_neg_eq_add,
-        mappingCone.ext_from_iff _ (p + 1) _ rfl, comp_add, mappingCone.inl_v_fst_v_assoc,
-        mappingCone.inl_v_snd_v_assoc, shiftFunctor_obj_X', sub_zero, add_zero, comp_id,
-        mappingCone.inr_f_fst_v_assoc, mappingCone.inr_f_snd_v_assoc, add_left_eq_self,
-        neg_eq_zero, true_and]
+    simp only [Cochain.ofHom_v, id_comp, comp_sub, sub_comp, assoc, reassoc_of% s_g,
+      ShortComplex.Splitting.s_r_assoc, ShortComplex.map_X₃, eval_obj, ShortComplex.map_X₁,
+      zero_comp, comp_zero, reassoc_of% f_r, zero_sub, sub_neg_eq_add,
+      mappingCone.ext_from_iff _ (p + 1) _ rfl, comp_add, mappingCone.inl_v_fst_v_assoc,
+      mappingCone.inl_v_snd_v_assoc, shiftFunctor_obj_X', sub_zero, add_zero, comp_id,
+      mappingCone.inr_f_fst_v_assoc, mappingCone.inr_f_snd_v_assoc, add_left_eq_self,
+      neg_eq_zero, true_and]
```

Comment:
Please do not break the `simp?` pattern, which facilitates the maintenance of the code.

Reply:
Ah, now it's timing out and breaking the build. What would you suggest here?

Reply:
Add `set_option maxHeartbeats 400000 in` before the declaration should make it work. (This increase is necessary only when it is compiled with `set_option says.verify true`, e.g. during CI)

Reply:
Hm, okay. It would be nice for these proofs to not use nonterminal simps